src/components/stats.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import './stats.css'
+import Vest from '../icons/gear_types/vest.png'
+import BreakdownItem from "./breakdownItem";
+
+function Stats() {
+    return (
+        <div className="statsGrid">
+            <div className="grid sectionSelectionGrid">
+                <img className="dsIco" src={Vest} onClick ="" alt="Weapon Talents"/>
+                <img className="dsIco" src={Vest} onClick ="" alt="Weapon Stats"/>
+                <img className="dsIco" src={Vest} onClick ="" alt="Offense"/>
+                <img className="dsIco" src={Vest} onClick ="" alt="Gear Talents"/>
+                <img className="dsIco" src={Vest} onClick ="" alt="Gear Brands/Sets"/>
+                <img className="dsIco" src={Vest} onClick ="" alt="Defense"/>
+                <img className="dsIco" src={Vest} onClick ="" alt="Skill Slot 1"/>
+                <img className="dsIco" src={Vest} onClick ="" alt="Skill Slot 2"/>
+            </div>
+            <div className="grid statValuesGrid">
+                <div id="weaponTalents" className="statSection">
+                    <h6>Weapon Talents</h6>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                </div>
+                <div id="weaponStats" className="statSection">
+                    <h6>Weapon Stats</h6>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                </div>
```

Comment:
I feel like this could  be a component which is repeated and just passed a name and the needed information.

Reply:
I have added this in the most recent commit. Haven't added the passing of needed info yet, as that will come in REACT3